### PR TITLE
Add catalog template for gitlab CI multi-runner

### PR DESCRIPTION
**Pull request by fredleger**

### CODE REVIEW

File: templates/gitlab-multi-runner/0/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+version: '2'
+
+services:
+
+  gitlab-runner-config:
+    image: gitlab/gitlab-runner:alpine
```
Comment (rawmind0) [inline templates/gitlab-multi-runner/0/docker-compose.yml]: Please, use concrete version instead implicit latest....Latest is no reproducible, please use gitlab/gitlab-runner:alpine-v9.3.0 or the needed version...

File: templates/gitlab-multi-runner/0/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    - docker:latest
+    - --docker-volumes
+    - /var/run/docker.sock:/var/run/docker.sock
+    - --docker-privileged
+    labels:
+      io.rancher.container.pull_image: always
+      io.rancher.container.start_once: 'true'
+  
+  gitlab-runner:
+    image: gitlab/gitlab-runner:alpine
```
Comment (rawmind0) [inline templates/gitlab-multi-runner/0/docker-compose.yml]: Please, use concrete version instead implicit latest....Latest is no reproducible, please use gitlab/gitlab-runner:alpine-v9.3.0 or the used version...

File: templates/gitlab-multi-runner/0/rancher-compose.yml [hunk 3]
```diff
@@ -0,0 +1,30 @@
+version: '2'
+
+catalog:
+  name: "gitlab-multi-runner"
+  version: "1.0.0"
```
Comment (rawmind0) [inline templates/gitlab-multi-runner/0/rancher-compose.yml]: Probably, version could be same as gitlab docker image version... 9.3.0 or used version...

File: templates/gitlab-multi-runner/config.yml [hunk 4]
```diff
@@ -0,0 +1,5 @@
+name: gitlab-multi-runner
+description: |
+  a Gitlab pipelines multi-runner, that will spawn privates runners in your infra.
+version: 1.0.0
```
Comment (rawmind0) [inline templates/gitlab-multi-runner/config.yml]: Probably, version could be same as gitlab docker image version... 9.3.0 or used version...

**Comment by rawmind0** [comment]
Hi @fredleger .... Thanks so much for the PR....

Could you please take a look at the change request??

**Comment by fredleger** [comment]
this should be fixed by now

**Comment by rawmind0** [comment]
Cool @fredleger thanks so much...

LGTM...

**Pull request merged**